tests/e2e/instrumentation-python-multicontainer/02-install-app.yaml

```diff
@@ -20,4 +20,4 @@ spec:
       - name: myapp
         image: ghcr.io/anuraaga/flask-hello-world:latest
```

Comment:
I know the problem is related to the Docker Hub images but... should we apply the same philosophy to all the images?